src/app/components/goods-card/goods-card.component.html

```diff
@@ -0,0 +1,12 @@
+<div class="goods-card">
+  <div class="header">{{ header }}</div>
+  <div class="image">
+    <a href="#" class="prev"></a>
+      <img [src]="images[0].url" />
+    <a href="#" class="next"></a>
+  </div>
+  <div class="common">
+    <p class="description">{{ description }}</p>
+    <button class="button" (click)="buyButtonClick.emit($event); $event.stopPropagation();">Купить</button>
```

Comment:
```suggestion
    <button
        class="button"
        (click)="buyButtonClick.emit($event); $event.stopPropagation();"
    >
        Купить
    </button>
```
Давай так подразнесем? И предлагаю $event.stopPropagation() унести в метод, чтобы шаблон сделать почище.